lib/asm-docs/arm32.ts

```diff
@@ -57,7 +57,7 @@ export class Arm32DocumentationProvider extends BaseAssemblyDocumentationProvide
         if (!isConditionalOpcode) {
             return null;
         }
-        const information = getAsmOpcode(isConditionalOpcode[1]);
+        const information = getAsmOpcode(isConditionalOpcode[1]) as AssemblyInstructionInfo;
```

Comment:
This line is unnecessary because information is already typed as any because it's a JS import.

Reply:
Without that casting, the command simply fails (at least on my machine)

Reply:
That's strange, are you running on main? `npm run ts-compile` runs fine on my machine on main